mne/io/artemis123/utils.py

```diff
@@ -82,3 +84,55 @@ def _compute_mne_loc(coil_loc):
     R = rotation3d_align_z_axis(z_axis)
     loc[3:13] = R.T.reshape(9)
     return loc
+
+
+def _read_pos(fname):
```

Comment:
Ill can do that to itemize the ctf issues, but maybe not for a few days.

Although I do think that a refactored  digitization class or structure should provide a way to read in digitized points from a file that supports the various use cases (including these two). and that might be the best time to think though these issues in the abstract. It is also a time where breaking backward comparability might be more understandable/palatable